ecs_model_deployer/src/lib/app.ts

```diff
@@ -0,0 +1,40 @@
+/**
+  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import { App } from 'aws-cdk-lib';
+import { LisaModelStack, LisaModelStackProps } from './lisa_model_stack';
+
+import { ConfigFile, ConfigSchema } from './schema';
+
+export const app = new App();
+
+const configFile = JSON.parse(process.env['LISA_CONFIG']!) as ConfigFile;
+const config = ConfigSchema.parse(configFile);
+
+const modelConfig = JSON.parse(process.env['LISA_MODEL_CONFIG']!);
+
+const stackProps: LisaModelStackProps = {
+    env: {
+        account: process.env['CDK_DEFAULT_ACCOUNT'],
+        region: process.env['CDK_DEFAULT_REGION']
```

Comment:
are these always guaranteed to be set?

Reply:
I believe so based on [this page](https://docs.aws.amazon.com/cdk/v2/guide/configure-env.html).
> The AWS CDK provides two environment variables that you can use within your CDK code: CDK_DEFAULT_ACCOUNT and CDK_DEFAULT_REGION. When you use these environment variables within the env property of your stack instance, you can pass environment information from your credentials and configuration files using the CDK CLI --profile option.

Reply:
From the same page, I'm gathering that these are still something that we'd have to intentionally define (otherwise how does it know what to set?)- this statement toward the bottom of the page makes me think that they aren't guaranteed to exist:
> These environment variables can override defaults at synthesis time if they exist

But I'm also assuming it worked in your account though, right? So maybe a non-issue?

Reply:
I believe the "if they exist" there is referring to the non-CDK provided `CDK_DEPLOY_ACCOUNT` and `CDK_DEPLOY_REGION`, and then the fallback is to used the CDK provided ones that I'm using. Yeah, it is working in my account, and my understanding is that it just gets set based on the account your `AWS_*` variables point to.